Polyform-Noncompete.md

```diff
@@ -28,25 +28,21 @@ The licensor grants you an additional copyright license to make changes and new
 
 The licensor grants you a patent license for the software that covers patent claims the licensor can license, or becomes able to license, that you would infringe by using the software.
 
-## Noncommercial Purposes
-
-Any noncommercial purpose is a permitted purpose.
-
-## Personal Uses
+## Fair Use
 
-Personal use for research, experiment, and testing for the benefit of public knowledge, personal study, private entertainment, hobby projects, amateur pursuits, or religious observance, without any anticipated commercial application, is use for a permitted purpose.
+You may have "fair use" rights for the software under the law. These terms do not limit them.
 
-## Noncommercial Organizations
+## Noncompete
 
-Use by any charitable organization, educational institution, public research organization, public safety or health organization, environmental protection organization, or government institution is use for a permitted purpose regardless of the source of funding or obligations resulting from the funding.
+Any purpose is a permitted purpose, except for providing any good or service that competes with the software or any good or service the licensor provides using the software.
```

Comment:
```suggestion
Any purpose is a permitted purpose, except for providing any good or service that competes with any good or service the licensor provides using the software.
```

Reply:
Is the software a good that the licensor provides using the software?

Reply:
Assuming it is a good the licensor provides, such as by offering licenses? However, here's another way to make it _slightly_ easier to see that what the licensor provides is the object of prohibited competition rather than the software being the object of prohibited competition (because what the latter means is not clear to me):

> Any purpose is a permitted purpose, except for providing any good or service that competes with the software or any **other** good or service the licensor provides using the software.

But I guess it's not a big deal either way.

Reply:
_other_ makes sense. Push a commit?

Reply:
#65